pkg/sentry/kernel/thread_group.go

```diff
@@ -516,6 +511,11 @@ func (tg *ThreadGroup) SetForegroundProcessGroup(tty *TTY, pgid ProcessGroupID)
 		return -1, linuxerr.EPERM
 	}
 
+	//if the calling process is a member of a background group, a SIGTTOU signal is sent to all members of this background process group.
```

Comment:
Formatting nit: comments are usually capitalized and wrapped at 80 characters, so this would be:
```go
  // If the calling process is a member of a background group, a SIGTTOU signal
  // is sent to all members of this background process group.
```